templates/firefox-ios.xliff

```diff
@@ -6278,6 +6282,34 @@ Sent from %2$@ 🦊 %3$@</source>
         <source>Close</source>
         <note>This is the swipe action title for closing an inactive tab by swiping, located in the Inactive Tabs section of the Tabs Tray</note>
       </trans-unit>
+      <trans-unit id="TabTray.TabsSelectorNormalTabsTitle.v140" xml:space="preserve">
+        <source>%@ Tabs</source>
+        <note>The title on the button to look at regular tabs. %@ is the number of regular tabs.</note>
+      </trans-unit>
+      <trans-unit id="TabTray.TabsSelectorSyncedTabsTitle.v140" xml:space="preserve">
+        <source>Sync</source>
+        <note>The title on the button to look at synced tabs.</note>
+      </trans-unit>
+      <trans-unit id="TabTrayCloseOldTabsTitle.v140" xml:space="preserve">
+        <source>Close Old Tabs…</source>
```

Comment:
Why is this capitalized versus next strings?

Reply:
That was capitalized to be consistent with the other option in the menu "Close All Tabs"

![Simulator Screenshot - iPhone 16 - 2025-05-07 at 11 09 13](https://github.com/user-attachments/assets/0fbe628d-b19e-4f27-b171-e133ab0b775e)